[2024-10-08T14:40:56.702+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-08T14:40:56.736+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_dag.analyze_data manual__2024-10-08T14:36:26.984993+00:00 [queued]>
[2024-10-08T14:40:56.757+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_dag.analyze_data manual__2024-10-08T14:36:26.984993+00:00 [queued]>
[2024-10-08T14:40:56.758+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-08T14:40:56.786+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): analyze_data> on 2024-10-08 14:36:26.984993+00:00
[2024-10-08T14:40:56.800+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=155) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T14:40:56.802+0000] {standard_task_runner.py:72} INFO - Started process 156 to run task
[2024-10-08T14:40:56.821+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_dag', 'analyze_data', 'manual__2024-10-08T14:36:26.984993+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpbfla_74n']
[2024-10-08T14:40:56.822+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask analyze_data
[2024-10-08T14:40:56.872+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline_dag.analyze_data manual__2024-10-08T14:36:26.984993+00:00 [running]> on host 24614c1ba665
[2024-10-08T14:40:56.965+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_dag' AIRFLOW_CTX_TASK_ID='analyze_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T14:36:26.984993+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T14:36:26.984993+00:00'
[2024-10-08T14:40:56.967+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-08T14:40:56.990+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-08T14:40:56.990+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/runtime.py /opt/***/scripts/analyze.py']
[2024-10-08T14:40:57.000+0000] {subprocess.py:86} INFO - Output:
[2024-10-08T14:40:57.189+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2024-10-08T14:40:57.190+0000] {subprocess.py:93} INFO -   File "/opt/***/scripts/analyze.py", line 2, in <module>
[2024-10-08T14:40:57.190+0000] {subprocess.py:93} INFO -     import dask.dataframe as dd
[2024-10-08T14:40:57.190+0000] {subprocess.py:93} INFO - ModuleNotFoundError: No module named 'dask'
[2024-10-08T14:40:58.166+0000] {subprocess.py:93} INFO - 
[2024-10-08T14:40:58.168+0000] {subprocess.py:93} INFO - Starting script: /opt/***/scripts/analyze.py
[2024-10-08T14:40:58.169+0000] {subprocess.py:93} INFO - Finished script: /opt/***/scripts/analyze.py
[2024-10-08T14:40:58.170+0000] {subprocess.py:93} INFO - Execution time: 1.01 seconds
[2024-10-08T14:40:58.171+0000] {subprocess.py:93} INFO - Average CPU usage: 0.00%
[2024-10-08T14:40:58.171+0000] {subprocess.py:93} INFO - Maximum RAM usage: 0.00 MB
[2024-10-08T14:40:58.172+0000] {subprocess.py:93} INFO - 
[2024-10-08T14:40:58.173+0000] {subprocess.py:93} INFO - ===== Summary =====
[2024-10-08T14:40:58.174+0000] {subprocess.py:93} INFO - Script: /opt/***/scripts/analyze.py
[2024-10-08T14:40:58.174+0000] {subprocess.py:93} INFO -   Execution time: 1.01 seconds
[2024-10-08T14:40:58.175+0000] {subprocess.py:93} INFO -   Average CPU usage: 0.00%
[2024-10-08T14:40:58.175+0000] {subprocess.py:93} INFO -   Maximum RAM usage: 0.00 MB
[2024-10-08T14:40:58.176+0000] {subprocess.py:93} INFO - 
[2024-10-08T14:40:58.176+0000] {subprocess.py:93} INFO - Total execution time for all scripts: 1.01 seconds
[2024-10-08T14:40:58.186+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-08T14:40:58.216+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-08T14:40:58.216+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline_dag, task_id=analyze_data, run_id=manual__2024-10-08T14:36:26.984993+00:00, execution_date=20241008T143626, start_date=20241008T144056, end_date=20241008T144058
[2024-10-08T14:40:58.247+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-08T14:40:58.266+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-08T14:40:58.267+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
